common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -55,6 +56,7 @@ def _open_editor(self):
         self.course_wiki_page.open_editor()
         self.course_wiki_edit_page.wait_for_page()
 
+    @pytest.mark.skip(reason='This test fails when using the new coursehome MFE.')
```

Comment:
@wittjeff Hi! I wanted to point out that with the coursehome mfe now as an opt-out, these accessibility tests failed. It seemed like the [new courseware flag PR](https://github.com/edx/edx-platform/pull/27757) also had a similar issue. Do you have any thoughts regarding these tests that are being skipped, and if we plan to comment these out, should we just delete them? 

tagging @Dillon-Dumesnil @ciduarte as well!